xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -580,25 +602,21 @@ void CRenderer::LoadSettings()
   m_subtitleHorizontalAlign = static_cast<SubtitleHorizontalAlign>(
       CServiceBroker::GetSettingsComponent()->GetSettings()->GetInt(
           CSettings::SETTING_SUBTITLES_CAPTIONSALIGN));
+  ResetSubtitlePosition();
 }
 
-int CRenderer::GetSubtitleVerticalMargin()
+double CRenderer::GetSubtitleVerticalMarginPerc()
 {
-  int subAlign = CServiceBroker::GetSettingsComponent()->GetSettings()->GetInt(
-      CSettings::SETTING_SUBTITLES_ALIGN);
+  // We return the vertical margin as percentage
+  // to fit the current screen resolution
+  const std::shared_ptr<CSettings> settings{CServiceBroker::GetSettingsComponent()->GetSettings()};
```

Comment:
Same for this one: Would be nice to wrap this as a getter in SubtitlesSettings to avoid spreading raw setting access everywhere (better encapsulation)